packages/@o3r/workspace/schematics/ng-add/helpers/linter.ts

```diff
@@ -1,23 +1,31 @@
-import { SchematicContext } from '@angular-devkit/schematics';
+import type { JsonObject } from '@angular-devkit/core';
+import type { SchematicContext, Tree } from '@angular-devkit/schematics';
 import { askConfirmation } from '@angular/cli/src/utilities/prompt';
 
+/**
+ * It the legacy ESLint config used in the repository
```

Comment:
```suggestion
 * If the legacy ESLint config is used in the repository
```
?
Also I would rename the function to make it more obvious that it's just a check
`useLegacyConfig` -> `isUsingLegacyConfig`